.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# How To Contribute
+To keep our repositories as uniform as possible, and work together effectively please consider the following:
+
+## Our Git Flow
+We use two primary branches.  `master` and `development`.  `development` is what you'll create pull requests against, and should be deployed to staging environments for testing. Once enough work has gone in to create a  deliverable piece value, and all testing is complete, we should merge `development` into `master`, which will then be deployed to a production environment.  Ideally `master` should always represent what is in production. 
+
```

Comment:
The merge into master only happens after a successful deployment. This is also missing the `version` tagging before deployment.